examples/jsm/renderers/common/Renderer.js

```diff
@@ -1048,10 +1168,28 @@ class Renderer {
 
 	}
 
+	_createObjectPipeline( object, material, scene, camera, lightsNode, passId ) {
+
+		const renderObject = this._objects.get( object, material, scene, camera, lightsNode, this._currentRenderContext, passId );
+
+		//
+
+		this._nodes.updateBefore( renderObject );
+
+		//
+
+		this._nodes.updateForRender( renderObject );
+		this._geometries.updateForRender( renderObject );
+		this._bindings.updateForRender( renderObject );
+
+		this._pipelines.getForRender( renderObject, this._compilationPromises );
+
+	}
 
 	get compute() {
 
-		return this.computeAsync;
+		console.warn( 'THREE.Renderer: compile() is deprecated and will be removed in r170, use compileAsync instead.' ); // @deprecated, r170
```

Comment:
I see that all other non-async methods just point to their async counterparts, why should we deprecate this one? /ping @sunag  @aardgoose @RenaudRohlinger